src/app/feedback/feedback-form/feedback-form.component.ts

```diff
@@ -49,8 +50,9 @@ export class FeedbackFormComponent implements OnInit {
   }
 
   private getHeaderText(): string {
-    const el = this.el.element.nativeElement.querySelector('h1:not([style*="display:none"]');
-    return el ? el.innerHTML : '';
+    // const el = this.el.element.nativeElement.querySelector('h1:not([style*="display:none"]');
```

Comment:
Should it be visible in the feedback window after submission? I couldn't see any header on the feedback in Chrome on master.

According to the MDN link it sounded like using a pseudoclass with `querySelector` would never return anything.